### PR TITLE
change installation id column

### PR DESCRIPTION
**Pull request by abiosoft**
https://digitalcrew.teamwork.com/#tasks/11630926

### CODE REVIEW

**Comment by Carpetsmoker** [comment]
It's consistent with the general scheme of `<table>_<col>` now?

**Comment by ready4god2513** [comment]
Yes, but we've standardized on `installationId` for that specific field.

File: auditMiddleware/audit.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ type Options struct {
 type Audit struct {
 	ID             int64  `db:"id"`
 	UserID         int64  `db:"users_id"`
-	InstallationID int64  `db:"installations_id"`
+	InstallationID int64  `db:"installationsId"`
```
Comment (ready4god2513) [inline auditMiddleware/audit.go]: Should be `installationId`

**Comment by Carpetsmoker** [comment]
Seems like an awful amount of effort to make things less consistent...

**Comment by PatrickWalker** [comment]
https://github.com/Teamwork/spaces-api/pull/76
is linked to this

**Comment by PatrickWalker** [comment]
@ready4god2513 this pull request fixes gordons rename request if you're happy to merge it

Thanks

**Comment by Felixls** [comment]
fixed on https://github.com/Teamwork/middleware/pull/33

**Pull request closed without merging**